voidcrew/modules/overmap/code/controllers/subsystem/overmap.dm

```diff
@@ -26,46 +28,74 @@ SUBSYSTEM_DEF(overmap)
 	///List of all simulated ships
 	var/list/simulated_ships = list()
 
+	#ifdef UNIT_TESTS
+	var/datum/map_template/shuttle/voidcrew/initial_ship_template = /datum/map_template/shuttle/voidcrew/box
+	#else
+	var/datum/map_template/shuttle/voidcrew/initial_ship_template = /datum/map_template/shuttle/voidcrew/box // TODO, remove this being set to something when shuttles generate their bounds correctly
+	#endif
+	var/obj/structure/overmap/ship/initial_ship
+
 /datum/controller/subsystem/overmap/Initialize(start_timeofday)
 	create_map()
 	setup_sun()
 	setup_dangers()
 	setup_planets()
-
-	return ..()
```

Comment:
Why don't we call parent anymore? It's used to know it's initialized and to log

Reply:
Not correct, please see `/datum/controller/master/proc/init_subsystem`